pytorch_to_returnn/converter/converter.py

```diff
@@ -156,6 +156,14 @@ def _make_tf_feed_dict(self, input: Data):
     else:
       for i, size in input.size_placeholder.items():
         d[size] = [self._inputs_np.shape[input.get_batch_axis(i)]] * n_batch  # not so relevant
+    if out_entry:
+      def _output_from_non_deterministic(tensor_entry_: TensorEntry):
+        return not all(call_.module.module.is_deterministic for call_ in tensor_entry_.output_from_calls)
+
+      for entry in [out_entry.output_from_calls[0].inputs_args[0]]:  # TODO: this is just the case I'm looking for right now, needs to be generalized
```

Comment:
TODO

Reply:
Why not `out_entry.output_from_calls` directly?

Reply:
The `RandInt` module is not in there, we have to do it recursively.
```
>>> out_entry.output_from_calls                                                                                                                                                                                                 
[<CallEntry 'add' <ModuleEntry <BinaryOperator>> (depth 0)>,
 <CallEntry 'output' <ModuleEntry <Copy>> (depth None)>]
```

I added a commit for this.